docs/ArgumentSetter-action.md

```diff
@@ -1,10 +1,37 @@
-# Driver Argument Setter Action
+# HTTP Argument Setter
 
 Description
 -----------
 
-Performs an HTTP request some endpoint to get a driver specification. Based on the spec,
-it will make another call to a nebula endpoint to get data about the dataset, which it will
-use to set 'input.path', 'input.properties', 'directives', and 'output.schema' arguments
-that can be used later on in the pipeline through macros.
+Performs an HTTP request to fetch arguments to set in the pipeline.
 
+This is most commonly used when the structure of a pipeline is static,
+and its configuration needs to be managed outside the pipeline itself.
+
+The HTTP response must return the arguments in a list:
+
+    {
+        "arguments" : [
+            { "name": "argument name", "value": "argument value" },
+            ...
+        ]
+    }
+
+Properties
+----------
+
+**URL:** URL to fetch arguments from.
+
+**HTTP Method:** HTTP method to use when making the request.
+
+**Request Body:** Body to use when making the request.
+
+**Connection Timeout:** Maximum amount of time in milliseconds to wait for a connection to be established.
```

Comment:
Should we document the defaults in our reference?

Reply:
We stopped documenting defaults and started just putting them as the pre-filled value in the field.